src/components/contact-us/contact-us-service.ts

```diff
@@ -77,6 +79,13 @@ export function contactUsService(
     htmlBody.push(`<span>[Session ID]</span>`);
     htmlBody.push(`<p>${optionalData.sessionId}</p>`);
 
+    htmlBody.push(`<span>[From page]</span>`);
+    if (!referer) {
```

Comment:
i would flip this round to be 

```
if (referer) {

htmlBody.push(`<p>${referer}</p>`);
    } else {
      htmlBody.push(`<p>undefined</p>`);
    }
```

As the message is for zendesk users i would change undefined to something a bit easier to understand like unable to capture referrer

Reply:
Updated